target/linux/ath79/dts/ar7161_netgear_wndr3800.dts

```diff
@@ -144,23 +164,24 @@
 			#address-cells = <1>;
 			#size-cells = <1>;
 
-			partition@0 {
+			uboot: partition@0 {
```

Comment:
beside the art label, none of them are really used/required. Please drop the unused labels.